docs/processing_information.md

```diff
@@ -21,7 +21,7 @@ We mapped reads to the transcriptome index using `salmon` with the default "sele
 Briefly, selective alignment uses a mapping score validated approach to identify maximal exact matches between reads and the provided index.
 For all samples, we used selective alignment to the `splici` index.
 
-A more detailed description of the mapping strategy invoked by `salmon` in conjunction with `alevin-fry` can be found in [Srivastava _et al._ (2020)](https://doi.org/10.1186/s13059-020-02151-8).
+A more detailed description of the mapping strategy invoked by `salmon` in conjunction with `alevin-fry` can be found in [Srivastava _et al._ 2020](https://doi.org/10.1186/s13059-020-02151-8).
```

Comment:
My internal style guide (based on Genetics) says to leave the date in parens here.

Reply:
The reason I changed this is because when I was adding the new reference I went to see what our preferred style was here and this was the only reference in this doc where we were using parenthesis. All the other references don't have them. Do we want to change all of them to have parenthesis then?

Reply:
The difference here is that `Srivastava _et al._` is part of the sentence that the reference appears in, so I only put the year in parenthesis. In the other cases, the entire reference is in parenthesis (except for the reference you added, where the names are again part of the sentence).